scripts/windows_compile.bat

```diff
@@ -97,26 +97,25 @@ if "%WEBEXTENSION_BUILD%" == "T" (
 ECHO Checking required commands...
 CALL :CheckCommand git
 CALL :CheckCommand python
-CALL :CheckCommand nmake
 CALL :CheckCommand cl
 CALL :CheckCommand qmake
 
 git submodule init
 git submodule update --remote --depth 1 i18n
 
 ECHO Copying the installer dependencies...
-CALL :CopyDependency libcrypto-1_1-x64.dll %BUILDDIR%\bin\libcrypto-1_1-x64.dll
-CALL :CopyDependency libssl-1_1-x64.dll %BUILDDIR%\bin\libssl-1_1-x64.dll
-CALL :CopyDependency libEGL.dll %BUILDDIR%\bin\libEGL.dll
-CALL :CopyDependency libGLESv2.dll %BUILDDIR%\bin\libGLESv2.dll
-CALL :CopyDependency Microsoft_VC142_CRT_x86.msm "%VCToolsRedistDir%\\MergeModules\\Microsoft_VC142_CRT_x86.msm"
-CALL :CopyDependency Microsoft_VC142_CRT_x64.msm "%VCToolsRedistDir%\\MergeModules\\Microsoft_VC142_CRT_x64.msm"
+CALL :CopyDependency libcrypto-1_1-x64.dll %BUILDDIR%\SSL\bin\libcrypto-1_1-x64.dll
+CALL :CopyDependency libssl-1_1-x64.dll %BUILDDIR%\SSL\bin\libssl-1_1-x64.dll
+
+ECHO "Checking vctools in %VCToolsRedistDir%"
+CALL :CopyDependency Microsoft_VC143_CRT_x86.msm "%VCToolsRedistDir%\\MergeModules\\Microsoft_VC143_CRT_x86.msm"
```

Comment:
This is a big change... Do we really need VC143?

Reply:
I think we _could_ v142 with an older version of msvc +vsstudio. 
Whole reason for this is, as qt6 needed a newer winsdk to compile, thus I upgraded my local vsstudio to vs2022 at the same time and [it does not seem to offer VC142  anymore](https://docs.microsoft.com/de-de/visualstudio/install/workload-component-id-vs-community?view=vs-2022). (In fetches/generate_vsstudio_bundle.ps1 you can check what we have on the vm) 
I did not do a check of "what is the lowest possible thing here" - as I didn't think of this to be a major change. Can you elaborate on this? (Also in case you got qt6-static-win building with vs2017, you can just get me your sdk-version and i'll update the dev-bundle)